File: src/main/java/com/ptah/entity/project/Project.java
package com.ptah.entity.project;

import com.ptah.common.BaseEntity;
import com.ptah.entity.contract.Participant;
import com.ptah.repository.userprofiling.ContactInfo;
import jakarta.persistence.Entity;
import jakarta.persistence.OneToOne;

@Entity
public class Project extends BaseEntity {
    private String name;
    private Participant owner;
    private ProjectStatus status;
    @OneToOne
    private ContactInfo contactInfo;
}

File: src/main/kotlin/com/ptah/entity/project/Project.kt
package com.ptah.entity.project

import com.ptah.common.BaseEntity
import com.ptah.entity.contract.Participant
import com.ptah.repository.userprofiling.ContactInfo
import jakarta.persistence.Entity
import jakarta.persistence.OneToOne

@Entity
class Project : BaseEntity() {
    private val name: String? = null

    @OneToOne
    private val owner: Participant? = null
    private val status: ProjectStatus? = null

    @OneToOne
    private val contactInfo: ContactInfo? = null
}
